pkg/cclient/README.md

```diff
@@ -0,0 +1,36 @@
+# Compiling the C Library
+
+## Creating the .h and the .so files:
+
+
+```go build -o lib_udsclient.so -buildmode=c-shared cclient.go```
+	- this creates .h and .so files from the cclient.go file.
+
+
+## Creating the .h and the .a files:
+
+```go build -o lib_udsclient.a -buildmode=c-archive ./cclient.go```
+	- this creates .h and .a files
+
+
+
+## Usage
+
+After creating the .h and/or the .a and the .o files as described in the previous steps,
+this is how to use the functions generated by cgo
+
+``` char* GetUdsClientVersion()```
```

Comment:
```suggestion
``` c
char* GetUdsClientVersion()
```
```

and the same for all the function signatures below Or make them a header

### char* GetUdsClientVersion()